addons/mail/static/src/models/discuss_sidebar_category/discuss_sidebar_category.js

```diff
@@ -79,6 +79,16 @@ registerModel({
             }
             return replace(categoryItems);
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeItemNewInputView() {
+            if (this.isAddingItem) {
+                return insertAndReplace();
+            }
+            return clear();
```

Comment:
`isAddingItem` is redundant now, we could just `insert`/`clear` the view directly in place of writing `true`/`false` on the boolean